database/migrations/2020_06_08_140727_update__s_k_config_in_production.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+use App\Evaluation;
+use App\Sponsor;
+use Illuminate\Database\Migrations\Migration;
+
+class UpdateSKConfigInProduction extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        // Only run in production
+        if (App::environment('production')) {
+            // Fetch the SK area
+            $sponsor = Sponsor::where('shortcode', 'SK')->first();
+            if (!$sponsor) {
```

Comment:
On staging no sponsor will be found. Check logs to verify this? As a test?

Reply:
yep